cartographer/mapping/map_builder.cc

```diff
@@ -279,7 +279,8 @@ std::map<int, int> MapBuilder::LoadState(
   // Set global poses of landmarks.
   for (const auto& landmark : pose_graph_proto.landmark_poses()) {
     pose_graph_->SetLandmarkPose(landmark.landmark_id(),
-                                 transform::ToRigid3(landmark.global_pose()));
+                                 transform::ToRigid3(landmark.global_pose()),
+                                 true);
```

Comment:
Here, `SetLandmarkPose()` is being called with `frozen = true`.  Shouldn't `MapBuilder::LoadState` apply the `load_frozen_state` flag to landmarks also?

Reply:
I guess it could, what I did here was just replicating the behavior that it had before.